zone/gm_commands/task.cpp

```diff
@@ -282,62 +266,45 @@ void command_task(Client *c, const Seperator *sep)
 
 		return;
 	} else if (is_show) {
-		c->ShowClientTasks(client_target);
+		target->ShowClientTasks(c);
 		return;
 	} else if (is_uncomplete) {
 		if (sep->IsNumber(2)) {
 			auto task_id = std::stoul(sep->arg[2]);
-			if (task_id && task_id < MAXTASKS) {
```

Comment:
Not sure why this changed

Reply:
The logic flow changed, the return comes first so we fail out instead of it coming after.